neo4j-examples/paradise-papers-django
# feature/19: create filter by Data Source

Closes [Ticket](https://trello.com/c/eGqFgh9e/19-as-a-user-i-want-a-feature-to-filter-results-of-the-search-so-that-i-can-see-specific-results-by-category)






## Testing

Steps to manually verify the change:
1. _Setup_: Runenvironment
2. _Exercise_: on the home page look at the filters.
3. _Verify_: Check if `filter by Data Source` is there. and check when you look at the filter it display the correct information
